test/check-openshift

```diff
@@ -89,4 +90,50 @@ class TestOpenshift(MachineCase):
         b.wait_present("svg line.DeploymentConfigReplicationController")
         b.wait_present("svg line.RouteService")
 
+    def testDelete(self):
+        m = self.machine
+        b = self.browser
+        b.wait_timeout(120)
+
+        self.login_and_go("/kubernetes")
+        b.wait_present("#service-list")
+
+        self.openshift.execute("oc create -f /tmp/mock-app-openshift.json")
+        b.wait_in_text("#service-list", "mock")
+
+        b.click("a[href='#/list']")
+        b.wait_present("#content .details-listing")
+        b.wait_present("#routes")
+        b.wait_present("#deployment-configs")
+
+        b.wait_present(".details-listing tbody[data-id='deploymentconfigs/default/frontend'] th")
+        b.wait_present(".details-listing tbody[data-id='routes/default/mock'] th")
+        self.assertEqual(b.is_present(".details-listing tbody[data-id='routes/default/mock'] th"), True)
+        self.assertEqual(b.is_present(".details-listing tbody[data-id='deploymentconfigs/default/frontend'] th"), True)
+
+        b.click(".details-listing tbody[data-id='routes/default/mock'] th")
+        b.click(".route-delete")
```

Comment:
@stefwalter this doesnt work
I get this error, Error: .route-delete is not visible
i logged in to the VM the class is applied to the button